src/core/ui.js

```diff
@@ -114,6 +114,58 @@ function createWarnButton(butName, arr, title) {
     ["label", `Document ${title.toLowerCase()}`],
   ]);
   ariaDecorate(button, ariaMap);
+
+
+  var numSteps = 20.0;
+  var boxElement
+  var prevRatio = 0.0;
+  var increasingColor = "rgba(255, 255, 255)";
+  var decreasingColor = "rgba(255, 255, 255, ratio)";
```

Comment:
Using inline arrows seems better IMHO.
```suggestion
  var decreasingColor = (ratio) => `rgba(255, 255, 255, ${ratio})`;
```

Reply:
As above... I wouldn't expect any CSS stuff here at all. All CSS transitions or animations should be handled in the CSS of file.